Translator-Bot Telegram

```diff
@@ -0,0 +1,149 @@
+
+mes = ['',0]
+
+@bot.message_handler() #Function for understanding the text
+def messager(message):
+	mes[0] = message.text
+	mes[1] = message.chat.id
+	text=message.text
+	if text.isdigit():
+	    bot.send_message(message.chat.id,"I don't understand you, write only text, please")
+	else:	
+	    bot.send_message(message.chat.id,'Choose language for translation',reply_markup=lang_btns)
+	    
+	    
+@bot.callback_query_handler(func=lambda call: True) #Function for translating
+def callback_query(call):
+	message_chat_id  = mes[1]
+	message_text = mes[0]
+	if call.data == 'ru':
+		translated_message = translator.translate(message_text,dest='ru')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id,text=message_send)
+
+	elif call.data == 'en':
+		translated_message = translator.translate(message_text,dest='en')
+		message_send =  {translated_message.text}
+		bot.send_message(message_chat_id,text=message_send)
+
+	elif call.data == 'ar':
+		translated_message = translator.translate(message_text,dest='ar')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)
+	elif call.data == 'zh-cn':
+		translated_message = translator.translate(message_text,dest='zh-cn')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)
+	elif call.data == 'fr':
+		translated_message = translator.translate(message_text,dest='fr')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)
+	elif call.data == 'de':
+		translated_message = translator.translate(message_text,dest='de')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)	
+	elif call.data == 'it':
+		translated_message = translator.translate(message_text,dest='it')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)
+	elif call.data == 'ja':
+		translated_message = translator.translate(message_text,dest='ja')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)	
+	elif call.data == 'pl':
+		translated_message = translator.translate(message_text,dest='pl')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)
+	elif call.data == 'pt':
+		translated_message = translator.translate(message_text,dest='pt')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)
+	elif call.data == 'es':
+		translated_message = translator.translate(message_text,dest='es')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)	
+	elif call.data == 'tr':
+		translated_message = translator.translate(message_text,dest='tr')
+		message_send = {translated_message.text}
+		bot.send_message(message_chat_id, text=message_send)	
```

Comment:
You can make this code significantly shorter just passing `call.data` to `dest`. If you what to check `call.data` is correct you can do it separately:
```
if call.data not in lang_dict:
    return
```